lib/storekit/client.rb

```diff
@@ -28,12 +28,13 @@ def verify!(base64_payload)
       resp = @http.request(req)
 
       decoded = JSON.parse resp.body
+      @last_response = ReceiptResponse.new(decoded)
```

Comment:
This is broken because whether the response is a receipt or not depends on the value of the `status` field.